src/ResourceManager/EventHub/Commands.EventHub.Test/SessionRecords/Microsoft.Azure.Commands.EventHub.Test.ScenarioTests.ConsumerGroupsTests/TestConsumerGroupsCRUD.json

```diff
@@ -0,0 +1,856 @@
+{
+  "Entries": [
+    {
```

Comment:
please reduce the directory name, we are increasing facing MAX_PATH issues in our repo.